src/xenia/gpu/vulkan/render_cache.cc

```diff
@@ -713,6 +716,28 @@ const RenderState* RenderCache::BeginRenderPass(VkCommandBuffer command_buffer,
   vkCmdBeginRenderPass(command_buffer, &render_pass_begin_info,
                        VK_SUBPASS_CONTENTS_INLINE);
 
+  // XXX HACK: clear the color target if the previous render pass was a
+  // depth-stencil pass targeting the same EDRAM tile
+  // (some games clear color targets with a depth pass, since it's slightly
+  // faster)
+  if (previous_render_pass && previous_render_pass->config.depth_stencil.used &&
```

Comment:
This may need to be axed for now. It's causing a regression in banjo-kazooie for me as of now.

Reply:
Good to know, I'll look at that game and see if I can fix the regression.